### DISCUSSION TITLE
Cannot download video from vimeo.com

### DISCUSSION THREAD

**Discussion by stanbird08** [aclap-dev/video-downloadhelper#1428] [Help and Q&A] [answered]
### Your question

I can download other videos; however it stop working on videos published after a particular date. 

The error message is "Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)."

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"94ad3c624fa84ee6"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:132.0) Gecko/20100101 Firefox/132.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "F:\\",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "F:\\"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "yt_warning": false,
    "open_count_store": 6,
    "successfull_dl": 87
}

**Comment by Mattmjs** [comment]
No help can be offered if you don't share where the video is.

**Reply by stanbird08** [reply, marked as answer]
> No help can be offered if you don't share where the video is.

Sorry! I should include the video link and it is working now. I found that clearing cookies and site data helped.
